addons/project/models/project.py

```diff
@@ -699,6 +699,28 @@ def unlink(self):
         analytic_accounts_to_delete.unlink()
         return result
 
+    @api.model
+    def _search(self, args, offset=0, limit=None, order=None, count=False, access_rights_uid=None):
+        new_order, item_index, desc = [], -1, False
+        for index, order_item in enumerate((order or self._order).split(',')):
+            order_item = order_item.strip().lower()
+            if order_item.startswith('is_favorite'):
```

Comment:
wouldn't it be funny if somebody added a field that starts with `is_favorite` like `is_favorite_2` or `is_favorite_desc` ?